src/mlpack/methods/ann/layer/add_impl.hpp

```diff
@@ -71,7 +71,7 @@ AddType<MatType>::operator=(AddType&& other)
 template<typename MatType>
 void AddType<MatType>::Forward(const MatType& input, MatType& output)
 {
-  output = input + arma::repmat(arma::vectorise(weights), 1, input.n_cols);
+  output = input + repmat(arma::vectorise(weights), 1, input.n_cols);
```

Comment:
You could change `arma::vectorise` to `vectorise` too, or in another PR, up to you. :+1:

Reply:
in another PR, will make them simple short and fast